pilot/pkg/bootstrap/server.go

```diff
@@ -1073,18 +1081,25 @@ func (s *Server) maybeCreateCA(caOpts *CAOptions) error {
 }
 
 // startCA starts the CA server if configured.
```

Comment:
Please update the comment to include RA.

Reply:
Will do